package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "axios": "1.7.7",
     "lodash": "4.17.21",
+    "moment": "2.30.1",
```

Comment:
Moment is a legacy project due to architectural issues, please choose another one - like luxon or day.js